hudi-utilities/src/main/java/org/apache/hudi/utilities/transform/ChainedTransformer.java

```diff
@@ -59,11 +65,15 @@ public ChainedTransformer(List<Transformer> transformersList) {
    * Creates a chained transformer using the input transformer class names. Refer {@link HoodieDeltaStreamer.Config#transformerClassNames}
    * for more information on how the transformers can be configured.
    *
-   * @param configuredTransformers List of configured transformer class names.
-   * @param ignore Added for avoiding two methods with same erasure. Ignored.
+   * @param sourceSchemaOpt                   Source Schema
```

Comment:
Can we add some context like "Schema from the dataset the transform is applied to"?